lib/util/cloudfront_signed_url.ex

```diff
@@ -0,0 +1,65 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule Antikythera.CloudfrontSignedUrl do
```

Comment:
I think `Antikythera.Aws.CloudfrontSignedUrl` is more appropriate than `Antikythera.CloudfrontSignedUrl`.
It is because CloudFront is one of AWS services.
(Changing module name breaks backwards compatibility. So I would like to avoid it.)

Reply:
I fixed it https://github.com/access-company/antikythera/pull/100/commits/24a3a21095fa3f47160f02f8caf7f2ff9af956e0